docs/release-notes/1264-connection-checks-for-dynamic-agents.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Improvements**
+
+- Add connection checks for dynamic agents. If a dynamically provisioned agent is not
```

Comment:
blocking: we should explain that it only terminates agents that are disconnected for a period of time and it ignores those agents that are starting up.

Reply:
done